charts/minecraft/values.yaml

```diff
@@ -127,6 +127,8 @@ minecraftServer:
   spawnMonsters: true
   # Determines if villagers will be spawned.
   spawnNPCs: true
+  # Sets the area that non-ops can not edit (0 to disable)
+  spawnProtection:  16
```

Comment:
I think the lint check is complaining about the extra space before 16

Reply:
fixed. thanks!